docs/concepts/data-views.asciidoc

```diff
@@ -32,16 +32,19 @@ uploaded a file, or added sample data,
 you get a data view for free, and can start exploring your data.
 If you loaded your own data, follow these steps to create a data view.
 
-. Open the main menu, then click to *Stack Management > Data Views*.
```

Comment:
A note about that: Currently it wouldn't work if there are no available data views, in this case you would need to go to Stack Management ... we should change this in the future